src/app/lib/analyticsUtils/index.js

```diff
@@ -151,6 +152,10 @@ export const getReferrer = (platform, origin, previousPath) => {
 export const getAtUserId = () => {
   if (!onClient()) return null;
 
+  // Users accessing the site on opera "extreme data saving mode" have the pages rendered by an intermediate service
+  // Attempting to track these users is just tracking that proxy, causing all opera mini visitors to have the same id
+  if (isOperaProxy()) return null;
```

Comment:
Shouldn't we return a uuid rather than null?

Reply:
When this is null, we don't pass an `idclient` to ATI

This seems to be what is expected when we're not trying to identify the user: https://developers.atinternet-solutions.com/general-en/craft-your-hit/

Reply:
Oh ok. Thanks